vendor/github.com/vmware/govmomi/govc/device/cdrom/insert.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright (c) 2014-2015 VMware, Inc. All Rights Reserved.
+Copyright (c) 2014-2016 VMware, Inc. All Rights Reserved.
```

Comment:
@emlin Why do some files have the copyright range from 2014-2017 and some have the copyright range from 2014-2016 like this one?  I'm just curious here.